city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,137 @@
+            paragraphs = paragraphs[1:]
+
+        for p in paragraphs:
+            t_name = ''
+            href = p.css('a::attr(href)').get()
+            t_list = p.css('*::text').getall()
+            dt_date = datetime.strptime(t_list[0], '%B %d, %Y')
+            try:
+                t_name = t_list[1]
+                if t_name == '':
+                    t_name = (split('-', split('/', href)[-1], 3)[-1])[:-4]
+            except IndexError:
+                pass
+
+            t_name = self.clean_up_title(t_name)
+            dt_two = dt_date.date()
+            min_list.append(Paragraph(link=href, date=dt_date, date2=dt_two, title=t_name))
+        return min_list
+
+    def _parse_links(self, item):
+        item_txt = ' '.join(item.css('*::text').extract()).strip()
+        item_txt = sub("Annual Meeting", "", item_txt)
+        item_txt = item_txt.replace("June", 'Jun').replace("Sept", 'Sep')
+
+        loc_items = split(',', item_txt, 4)
+        d_date = ''.join([loc_items[0], loc_items[1]]).replace('.', '')
+        dt = datetime.strptime(d_date, '%b %d %Y')
+        date_short = dt.date()
+        try:
+            records = list(filter(lambda d: d['date2'] == date_short, self.minutes_list))
+            the_link = records[0]['link']
+        except IndexError:
+            the_link = ''
+        return the_link
+
+    def _parse_title(self, item):
+        if "Annual Meeting" in ''.join(item.css("p::text").getall()):
+            return "Annual Meeting"
+        return COMMISSION
+
+    def _parse_location(self):
+        commission_loc = {
+            "name": "Sheil Park",
+            "address": "3505 N. Southport Ave., Chicago, IL 60657",
+        }
+        # committee_loc = {
+        #     "name": "Lakeview Chamber of Commerce",
+        #     "address": "1409 W. Addison St. in Chicago",
+        # }
+        # if self.committee_type(item):
+        #     return committee_loc
+        return commission_loc
+
+    def _parse_start(self, item):
+        item_txt = ' '.join(item.css('*::text').extract()).strip()
+        item_txt = sub("Annual Meeting", "", item_txt)
+        item_txt = item_txt.replace("June", 'Jun').replace("Sept", 'Sep')
+        p_idx = max(item_txt.find('am'), item_txt.find('pm'), 0) + 2  # so we can slice
+        front_str = item_txt[:p_idx]  # strip rest of the string
+        time_str = front_str.replace("am", "AM").replace("pm", "PM").replace('.', '')
+        return datetime.strptime(time_str, '%b %d, %Y, %H:%M %p')
+
+    def _validate_locations(self, response):
+        commission_path = "#content-232764 div.panel-body > p:nth-child(1) > strong::text"
+        commission_addy = response.css(commission_path).get()
+        commission_result = commission_addy.find("Sheil")
+        committee_addy = response.css("#content-238036 div.panel-body p em::text").get()
+        committee_result = committee_addy.find("Chamber")
+
+        if commission_result < 0:  # fail
```

Comment:
You can simplify this conditional by combining the two conditions here and just returning "Meeting location has changed", since the main purpose of this is just to create an error in Sentry that we look into

Reply:
Thanks for all your comments Patrick!  Really appreciate your dedication to this project.